website/views.py

```diff
@@ -283,22 +284,32 @@ def dashboard(auth):
 @must_be_logged_in
 def watched_logs_get(**kwargs):
     user = kwargs['auth'].user
-    page_num = int(request.args.get('pageNum', '').strip('/') or 0)
-    page_size = 10
-    offset = page_num * page_size
-    recent_log_ids = itertools.islice(user.get_recent_log_ids(), offset, offset + page_size + 1)
+    try:
+        page = int(request.args.get('page', 0))
+    except ValueError:
+        raise HTTPError(http.BAD_REQUEST, data=dict(
+            message_long='Invalid value for "page".'
+        ))
+    try:
+        size = int(request.args.get('size', 10))
+    except ValueError:
+        raise HTTPError(http.BAD_REQUEST, data=dict(
+            message_long='Invalid value for "size".'
+        ))
+
+    start = page * size
+    total = sum(1 for x in user.get_recent_log_ids())
+    recent_log_ids = itertools.islice(user.get_recent_log_ids(), start, start + size)
```

Comment:
This is duplicated with the contributor pagination logic? Can we DRY this up somehow?